src/passes/strings-encoding/StringEncoding.cpp

```diff
@@ -553,30 +556,56 @@ static Expected<std::unique_ptr<llvm::Module>> loadModule(StringRef Path,
   return M;
 }
 
+static Expected<std::unique_ptr<llvm::Module>>
+generateModule(StringRef Routine, StringRef Triple, LLVMContext &Ctx,
+               ArrayRef<std::string> ExtraArgs) {
+  llvm::hash_code HashValue = llvm::hash_combine(Routine, Triple);
+  const std::string &IRModuleFilename =
+      "/tmp/module_" + std::to_string(HashValue) + ".ll";
```

Comment:
We should have a dedicated prefix like `omvll-` and maybe we want to use the LLVM helper functions to create temporary files (it works reliably cross-platform). See e.g.: `llvm::sys::fs::createTemporaryFile()`